docs/guides/clawback-primitive-guide.md

```diff
@@ -8,60 +8,64 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 ```
 
-[notes
-synced full node and synced wallet are both required
-
-]
-
-
 ## Intro
 
-This document will show you how to use Chia's clawback primitive. Clawback currently is implemented as a command-line tool. Wallet developers are welcome to integrate it into their GUI wallets. For additional technical resources, see the following:
+This document will show you how to use Chia's clawback primitive. Clawback currently is implemented as a command-line tool only. Wallet developers are welcome to integrate it into their GUI wallets. For additional technical resources, see the following:
```

Comment:
we've been calling this a developer primitive, so we might want to explain that it doesn't implement wallet functionality to handle incoming clawbacks and resync deleted coin stores. Rather it's for developers to understand the process of how clawbacks work. Also that we'll have a user-friendly implementation in our wallet client in a future release